tests/manage/mcg/test_endpoint_autoscale.py

```diff
@@ -27,47 +31,61 @@ class TestEndpointAutoScale(MCGTest):
     # with an autoscaling conifguration of 1-2
     MIN_ENDPOINT_COUNT = 1
     MAX_ENDPOINT_COUNT = 2
+    options = {
+        "create": [
+            ("name", "job1"),
+            ("name", "job2"),
+            ("name", "job3"),
+            ("runtime", "1200"),
+        ],
+        "job1": [
+            ("iodepth", "4"),
+            ("rw", "randrw"),
+            ("bs", "32k"),
+            ("size", "64m"),
+            ("numjobs", "4"),
+        ],
+        "job2": [
+            ("iodepth", "16"),
+            ("rw", "randrw"),
+            ("bs", "64k"),
+            ("size", "512m"),
+            ("numjobs", "4"),
+        ],
+        "job3": [
+            ("iodepth", "32"),
+            ("rw", "randrw"),
+            ("bs", "128k"),
+            ("size", "1024m"),
+            ("numjobs", "4"),
+        ],
+    }
 
     def test_scaling_under_load(self, mcg_job_factory):
-        self._assert_endpoint_count(1)
-
-        options = {
-            "create": [
-                ("name", "job1"),
-                ("name", "job2"),
-                ("name", "job3"),
-                ("runtime", "1200"),
-            ],
-            "job1": [
-                ("iodepth", "4"),
-                ("rw", "randrw"),
-                ("bs", "32k"),
-                ("size", "64m"),
-                ("numjobs", "4"),
-            ],
-            "job2": [
-                ("iodepth", "16"),
-                ("rw", "randrw"),
-                ("bs", "64k"),
-                ("size", "512m"),
-                ("numjobs", "4"),
-            ],
-            "job3": [
-                ("iodepth", "32"),
-                ("rw", "randrw"),
-                ("bs", "128k"),
-                ("size", "1024m"),
-                ("numjobs", "4"),
-            ],
-        }
-        for i in range(10):
-            exec(f"job{i} = mcg_job_factory(custom_options={options})")
-        self._assert_endpoint_count(2)
-
-        for i in range(10):
-            exec(f"job{i}.delete()")
-            exec(f"job{i}.ocp.wait_for_delete(resource_name=job{i}.name, timeout=60)")
-        self._assert_endpoint_count(1)
+        self._assert_endpoint_count(self.MIN_ENDPOINT_COUNT)
+        endpoint_cnt = get_endpoint_pod_count(config.ENV_DATA["cluster_namespace"])
+        get_hpa_utilization(config.ENV_DATA["cluster_namespace"])
```

Comment:
Method is not accepting returned value here. Please remove this line if not needed

Reply:
Can you clarify?  MIN_ENDPOINT_COUNT and MAX_ENDPOINT_COUNT are used to pass the value to check for endpoint count.

Reply:
I am referring `get_hpa_utilization(config.ENV_DATA["cluster_namespace"])` code here

Reply:
get_hpa_utilization(config.ENV_DATA["cluster_namespace"] is used to show the info of hpa.  It doesn't need to return value.  I don't think this need to remove.